python/opentracing/Dockerfile.server

```diff
@@ -5,7 +5,7 @@ WORKDIR /app
 ADD requirements.txt .
 RUN pip install -r requirements.txt
 
-RUN opentelemetry-bootstrap -a install
+# RUN opentelemetry-bootstrap -a install
```

Comment:
Wouldn't it be better to just remove this line?

Reply:
usually this works fine, which is why i was planning on leaving it there and re-enabling it after the next release of launcher